code/data_processor_block/README.md

```diff
@@ -0,0 +1,26 @@
+# Data Processor Block
+#### Takes packets with data from base_parser and sends it to graphyte
+### Run it:
+`python data_processor.py config.ini`
+### Config File:
+`config.ini` - You specify here your graphite address (if you host it on the same device its `localhost`), graphyte prefix (kinda like folder with your data in graphyte) and sleep_time  
```

Comment:
It's `Graphite` not `Graphyte` 🤣

Reply:
Aye DJ Evening-Morgen, but how do you exblain that python lib is called `graphyte` ? Hmm? Cocoa Clogged?

Reply:
Here we go again,
`Graphite` - metric store engine
`Graphyte` - Python module name
So when you `prefix` metrics, you prefix them in datastore or library itself? I bet it is the first one. Probably for the simplicity sake guys from `graphyte` called it just [`prefix`](https://github.com/Jetsetter/graphyte/blob/master/graphyte.py#L30)
And when it comes to defining tree (folder) structure you definetly do that in `Graphite` :P
So something is no yes here, cocoa not clogged at all!

Reply:
Okay, cosmetic fix, check out 0fbbb63